pkg/reconciler/apis/apibinding/apibinding_controller.go

```diff
@@ -267,20 +270,25 @@ func (c *controller) enqueueAPIExport(obj interface{}, logSuffix string) {
 	}
 
 	for _, obj := range bindingsForExport {
-		c.enqueueAPIBinding(obj, fmt.Sprintf(" because of APIExport %s%s", key, logSuffix))
+		c.enqueueAPIBinding(obj, logger.WithValues("APIExport", key), fmt.Sprintf(" because of APIExport%s", logSuffix))
 	}
 }
 
 // enqueueCRD maps a CRD to APIResourceSchema for enqueuing.
-func (c *controller) enqueueCRD(obj interface{}) {
+func (c *controller) enqueueCRD(obj interface{}, logger logr.Logger) {
 	crd, ok := obj.(*apiextensionsv1.CustomResourceDefinition)
 	if !ok {
 		runtime.HandleError(fmt.Errorf("obj is supposed to be a CustomResourceDefinition, but is %T", obj))
 		return
 	}
+	logger = logger.WithValues(
+		"CustomResource", fmt.Sprintf("%s.%s", crd.Spec.Names.Plural, crd.Spec.Group),
+		"CustomResourceDefinition", logging.Key(crd),
```

Comment:
as written above, am not convinced having keys like that has value, or at least log the key **and** the individual ws+ns+name values in some way. Maybe also the kind and group?

Reply:
https://github.com/kcp-dev/kcp/pull/1695#discussion_r939332761